src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -164,10 +198,14 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
           case (Some(r1), Some(r2)) =>
             Seq(createSamRecord(r1, FirstOfPair), createSamRecord(r2, SecondOfPair))
           case _                    =>
-            rejectRecords(recs)
```

Comment:
This could double count records that were rejected for being fragment reads or for having a minority cigar.